plugins/outputs/logzio/logzio.go

```diff
@@ -40,15 +40,16 @@ type Logzio struct {
 	Token   string            `toml:"token"`
 	URL     string            `toml:"url"`
 	Timeout internal.Duration `toml:"timeout"`
+	tls.ClientConfig
```

Comment:
Since you are adding this, don't forget to update the sample config and readme with [the new options](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/http/http.go#L65-L70)

Reply:
Sure :)